src/plays/dynamic-routes/DynamicRoutes.jsx

```diff
@@ -8,12 +8,12 @@ import { useEffect, useState } from "react";
 function DynamicRoutes(props) {
 
   // Your Code Start below.
-  let { param } = useParams(); // return the parameter of url
+  let { param1 } = useParams(); // return the parameter of url
```

Comment:
@koustov why was it renamed to param1?

Reply:
In routedef.jsx, its called param1 and param2. Need to be in sync